_pages/integrations/android-dl.md

```diff
@@ -4,7 +4,7 @@ layout: article
 title: "Android (Deep Learning)"
 categories: app-development
 tags: [integrations, app-development]
-permalink: integrations/android-dl
+permalink: integrations/android-dl/
```

Comment:
still some inconsistency with the leading slash on these paths.

Reply:
from your template PR
"- `permalink` and `redirect_from/to` properties have `/` at beginning and end (e.g., `/path/`)"

Reply:
thanks fixed all the ones i could find, commit incoming